taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -87,19 +87,18 @@ def __init__(self, projectid, sampleid, **kwargs):
         self.reportpath = getattr(self, 'reportpath', None)
         self.force = getattr(self, 'force', False)
         self.stage_only = getattr(self, 'stage_only', False)
+        self.projectname = None
         #Fetches a project name, should always be availble; but is not a requirement
         try:
-            n = db.project_entry(db.dbcon(), projectid)['name']
-            self.projectname = n
+            self.projectname = getattr(db.project_entry(db.dbcon(), projectid), 'name')
```

Comment:
I'd prefer to just use `self.projectname = getattr(db.project_entry(db.dbcon(), projectid), 'name', None)`, that will set projectname to None if the `name` attribute does not exist.  Then you don't need to explicitly initialize `self.projectname` or wrap it in a `try ... except` clause. If you still want to do that, you need to catch an `AttributeError` rather than a `KeyError`